template_project/app/build.gradle

```diff
@@ -148,7 +148,7 @@ task generateShorcutsFile {
                     'intent'(
                             'android:action': 'android.intent.action.MAIN',
                             'android:targetPackage': twaManifest.applicationId,
-                            'android:targetClass': 'android.support.customtabs.trusted.LauncherActivity',
+                            'android:targetClass': 'com.google.androidbrowserhelper.trusted.LauncherActivity',
```

Comment:
Are you sure about this? I'm guessing you tested it and the previous targetClass did not work.

Reply:
I think this is a left-over when we migrated to `androix.browser-1.2.0`, which doesn't have a LauncherActivity anymore. This should have been changed with the other references across the code.